.gitignore

```diff
@@ -217,3 +217,4 @@ pip-log.txt
 
 sftp-config.json
 node_modules/
+package-lock.json
```

Comment:
This file should be committed to github.

Reply:
done